### ISSUE TITLE
Implement SVGP for US Flight data

### ISSUE THREAD

**Issue by Corleno** [GPflow/GPflow#1022]
I implement SVGP model for the US flight data mentioned in the Hensman 2014 using the number of inducing point = 100, batch_size = 1000, learning rate = 1e-5 and maxiter = 500. 
The result is pretty strange end ELBO does not increase as shown in 

# Inference using SVI
M = 100
D = 8
def init():
    kern = gpflow.kernels.RBF(D, 1, ARD=True)
    Z = X_train[:M, :].copy()
    m = gpflow.models.SVGP(X_train, Y_train.reshape([-1,1]), kern, gpflow.likelihoods.Gaussian(), Z, minibatch_size=1000)
    return m
m = init()

m.feature.trainable = True
opt = gpflow.train.AdamOptimizer(learning_rate = 0.00001)
m.compile()
opt.minimize(m, step_callback=logger, maxiter = 500)

plt.plot(logf)
plt.xlabel('iteration')
plt.ylabel('ELBO')
m

  | class | prior | transform | trainable | shape | fixed_shape | value-- | -- | -- | -- | -- | -- | -- | --
Parameter | None | (none) | True | (100, 8) | True | [[1.0, 19.0, 6.0, 558.0, 850.0, 90.0, 651.0, 7...
Parameter | None | +ve | True | (8,) | True | [0.999353586485411, 1.0002227184179315, 0.9996...
Parameter | None | +ve | True | () | True | 0.996846169909223....
Parameter | None | +ve | True | () | True | 1.003136517167067....
Parameter | None | (none) | True | (100, 1) | True | [[0.0017896087469726743],.... 
Parameter | None | LoTri->vec | True | (1, 100, 100) | True | [[[0.9982137508494925, ...

![svgp_flight](https://user-images.githubusercontent.com/11898466/63064759-b7981200-beb7-11e9-8852-292c091afaed.png)

**Comment by awav** [comment]
Hello @Corleno,

Could you please re-submit this question on StackOverflow?

<img width="733" alt="Screenshot 2019-08-15 at 11 44 05" src="https://user-images.githubusercontent.com/24483645/63090173-60934b00-bf52-11e9-8ddf-c33bdad0965e.png">

Thanks!

**Comment by Corleno** [comment]
Hi Artem,

I think this is the issue caused by the Adam optimization. Could you please
give me more information about how you implement the optimization in gpflow?
Thanks,

Rui
